ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -40,3 +43,30 @@ compute:
     azure:
       type: ${COMPUTE_NODE_TYPE}
 EOF
+else
+  cat >> "${CONFIG}" << EOF
+baseDomain: ${BASE_DOMAIN}
+platform:
+  azure:
+    baseDomainResourceGroupName: os4-common
+    region: ${REGION}
+controlPlane:
+  name: master
+  platform:
+    azure:
+      type: ${master_type}
+compute:
+- name: worker
+  replicas: ${workers}
+  platform:
+    azure:
+      type: ${COMPUTE_NODE_TYPE}
+EOF
+fi
+
+# Check if proxy is set
```

Comment:
In this step, no need proxy yet, the proxy will be used in `ipi-install-install-commands.sh`

Reply:
yes, see https://github.com/openshift/release/pull/26774/files#r823687725
just remove L67-72